app/uk/gov/hmrc/customs/declarations/information/xml/BackendPayloadCreator.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.declarations.information.xml
+
+import javax.inject.Singleton
+import org.joda.time.DateTime
+import uk.gov.hmrc.customs.declarations.information.model._
+import uk.gov.hmrc.customs.declarations.information.model.actionbuilders.AuthorisedRequest
+
+import scala.xml.{NodeSeq, Text}
+
+@Singleton
+class BackendPayloadCreator() {
+
+  private val newLineAndIndentation = "\n        "
+
+  def create[A](correlationId: CorrelationId,
+                date: DateTime,
+                searchType: SearchType,
+                maybeApiSubscriptionFieldsResponse: Option[ApiSubscriptionFieldsResponse])
+               (implicit asr: AuthorisedRequest[A]): NodeSeq = {
+
+    val searchElement = <n1:labelToRename>{searchType.toString}</n1:labelToRename>.copy(label = searchType.label)
+
+    <n1:queryDeclarationStatusRequest
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:n1="http://gov.uk/customs/declarationInformationRetrieval/status/v2"
+    xsi:schemaLocation="http://gov.uk/customs/declarationInformationRetrieval/status/v2 queryDeclarationStatusRequest.xsd">
+      <n1:requestCommon>
+        {Seq[NodeSeq](
+        <n1:clientID>99999999-9999-9999-9999-999999999999</n1:clientID>, Text(newLineAndIndentation),
```

Comment:
Need to pull this out of ASF for CSP and dummy for non-CSP - also does it need to be a VALID UUID?

Reply:
Both Errol and John confirmed that neither MDG nor DIS use this value. So for simplicity I left it at a fixed value for both. Yes, it does need to be a uuid. Schema mandates that.